recipe/meta.yaml

```diff
@@ -46,5 +121,6 @@ about:
   summary: zlib data compression library for the next generation systems
 
 extra:
+  feedstock-name: zlib-ng
   recipe-maintainers:
     - hmaarrfk
```

Comment:
At this stage, I cannot be the only one to maintain this, i had been ok being "alone" since others were "opting in" to the "riskyness" of zlib-ng, but now, it is really beyond me.